modules/lightning_features/lightning_core/lightning_core.module

```diff
@@ -16,7 +15,13 @@ use Drupal\lightning_core\Entity\Role;
 use Drupal\lightning_core\Form\RoleForm;
 use Drupal\lightning_core\OverrideHelper as Override;
 use Drupal\lightning_core\Plugin\views\filter\Bundle;
-use Drupal\path\Plugin\Field\FieldType\PathFieldItemList;
+
+/**
+ * Implements hook_local_tasks_alter().
+ */
+function lightning_core_local_tasks_alter(array &$local_tasks) {
+  $local_tasks['entity.node.edit_form']['options']['attributes']['rel'] = 'edit-form';
```

Comment:
This is a nicety for testing -- it lets us click the actual Edit tab for a node, regardless of what its text says.